www/src/assets/icons/layer-icon.js

```diff
@@ -107,7 +107,7 @@ const LayerIcon = ({ name, fillColor = colors.grey[50] }) => {
         />
       </svg>
     ),
-    Data: (
+    GraphqlLogo: (
```

Comment:
I feel like `GraphQLLogo` makes sense here but by no means should that hold up the PR.

Reply:
Maybe GraphQlLogo? Idk, it all seems not ideal!

Reply:
Ooh, maybe just leave it alone. It's not terrible! 😅